verificarlo.in.in

```diff
@@ -131,7 +131,7 @@ def linker_mode(sources, options, output, args):
     extra_args += "-DDDEBUG " if args.ddebug else ""
     extra_args += "-DINST_FUNC " if args.inst_func else ""
 
-    shell('{clang} -c -O3 -Wno-varargs {extra_args} -o .vfcwrapper.o {vfcwrapper} -I {mcalib_includes}'.format(
+    shell('{clang} -c -O3 -march=native -Wno-varargs {extra_args} -o .vfcwrapper.o {vfcwrapper} -I {mcalib_includes}'.format(
```

Comment:
It is just here to patch partially vectors problem, but it may be remove after yohan PR.